pkg/manifests/manifests.go

```diff
@@ -60,7 +60,7 @@ const (
 	defaultIgnitionVersion       = "3.2.0"
 	defaultIgnitionContentSource = "data:text/plain;charset=utf-8;base64"
 	defaultOCIHooksDir           = "/etc/containers/oci/hooks.d"
-	defaultScriptsDir            = "/usr/bin"
+	defaultScriptsDir            = "/usr/local/bin"
```

Comment:
doesn't really matter, so I'm OK with this

Reply:
looks like it matters, it tried to write it before to `/usr/bin`, and the machine config said that it is the read-only directory.

Reply:
ok, good point. I was thinking only from the FileSystem Hierarchy Standard perspective.

Reply:
same, this deserves a comment above `defaultScriptsDir`